modules/conflict_resolver/jsx/conflictResolverIndex.js

```diff
@@ -0,0 +1,297 @@
+            Data: data,
+            isLoaded: true,
+          });
+        }).catch((error) => {
+          // console.log('error: ' + error);
+    });
+  }
+
+  updateFilter(filter) {
+    this.setState({filter});
+    if (this.child !== undefined) {
+      this.child.updateFilterState(filter);
+    }
+  }
+
+  resetFilters() {
+    this.filter.clearFilter();
+  }
+
+  handleSubmit(e) {
+    e.preventDefault();
+  }
+
+  // Render the HTML
+  render() {
+    // Waiting for async data to load
+    if (!this.state.isLoaded) {
+      return <Loader/>;
+    }
+
+    const tabs = [];
+    tabs.push(
+      <UnresolvedConflictsPane
+        TabId='UnresolvedConflicts'
+        key={1}
+        url={this.props.url}
+        data={this.state.Data}
+        ref={(instance) => {
+          this.child = instance;
+        }}
+      />
+    );
+    const tabList = [
+      {
+        id: 'UnresolvedConflicts',
+        label: 'Unresolved Conflicts',
+      },
+      {
+        id: 'ResolvedConflicts',
+        label: 'Resolved Conflicts',
+      },
+    ];
+
+    // Hide empty options for Select Elements
+    if (this.state.Data) {
+      if (this.state.Data.form.site) {
+        this.state.Data.form.site.emptyOption = false;
+      }
+      if (this.state.Data.form.instrument) {
+        this.state.Data.form.instrument.emptyOption = true;
+      }
+      if (this.state.Data.form.visitLabel) {
+        this.state.Data.form.visitLabel.emptyOption = false;
+      }
+    }
+
+    return (
+      <div>
+        <div>
+          <FilterForm
+            Module='conflict_resolver'
+            id='conflict_resolver_filter'
+            name='conflict_resolver_filter'
+            columns={2}
+            ref={this.setFilterRef}
+            formElements={this.state.Data.form}
+            onUpdate={this.updateFilter}
+            filter={this.state.filter}
+          >
+            <ButtonElement
+              id='testClearForm1'
+              label='Clear Filters'
+              type='reset'
+              onUserInput={this.resetFilters}
+            />
+          </FilterForm>
+        </div>
+        <div>
+          <Tabs tabs={tabList} defaultTab='UnresolvedConflicts'>
+            {tabs}
+          </Tabs>
+        </div>
+      </div>
+    );
+  }
+}
+ConflictResolverApp.propTypes = {
+  url: PropTypes.object.isRequired,
+};
+ConflictResolverApp.defaultProps = {
+  url: {
+    base: '',
+    data: {
+      unresolved: '',
+      resolved: '',
+    },
+  },
+};
+
+/**
+ * Render conflictResolver on page load
+ */
+window.addEventListener('load', () => {
+  ReactDOM.render(
+    <ConflictResolverApp
+      url={{
+        base: loris.BaseURL,
+        data: {
+          unresolved: loris.BaseURL + '/conflict_resolver/?format=json',
+          resolved: loris.BaseURL + '/conflict_resolver/resolved_conflicts/?format=json',
+        },
+      }}
+    />,
+    document.getElementById('lorisworkspace')
+  );
+  // Prevent tab switching
```

Comment:
Why would you want a second tab that can't be switched to?

Reply:
@driusan The tab switching of the react component doesn't correlate with how the backend was designed. The backend db would get queried when switching tabs before react and basically each tab is a link that retrieves the conflicts or resolved depending on the tab clicked. The part you're commenting on disables just a div hide/show functionality of the react component and shouldn't be done because it makes the data required to be retrieved even larger.